### PR TITLE
Add ability to specify a custom configuration file

### PR DESCRIPTION
**Pull request by guilhem**
This is useful when bootstraping a new workstation.

### CODE REVIEW

File: mackup/mackup.py [hunk 1]
```diff
@@ -19,9 +19,9 @@ class Mackup(object):
 
     """Main Mackup class."""
 
-    def __init__(self):
+    def __init__(self, filename=None):
```
Comment (lra) [inline mackup/mackup.py]: Can you assert that the filename parameter is valid? e.g. https://github.com/lra/mackup/blob/master/mackup/config.py#L40

File: mackup/main.py [hunk 2]
```diff
@@ -12,11 +12,12 @@
   mackup --version
 
 Options:
-  -h --help     Show this screen.
-  -f --force    Force every question asked to be answered with "Yes".
-  -n --dry-run  Show steps without executing.
-  -v --verbose  Show additional details.
-  --version     Show version.
+  -h --help               Show this screen.
+  -c FILE --config=FILE   Configuration file to use.
```
Comment (lra) [inline mackup/main.py]: If we go down this road, I guess we could leverage docopt to set a default file in the docstring, e.g. https://github.com/docopt/docopt#help-message-format

**Comment by simobasso** [comment]
Hi @guilhem @lra , any update on this one?

**Comment by guilhem** [comment]
Close in favor of #1254

**Pull request closed without merging**